connect/client_test.go

```diff
@@ -0,0 +1,56 @@
+package connect_test
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	cluster_api "github.com/pomerium/zero-sdk/cluster"
+	"github.com/pomerium/zero-sdk/connect"
+	"github.com/pomerium/zero-sdk/token"
+)
+
+func TestConnectClient(t *testing.T) {
```

Comment:
Just so I understand: it looks like this test is intended more for use during development than for automated testing; is that accurate?

Reply:
correct. I probably need to add an actual isolated test here.

Reply:
actually no, there would be too many things to mock to make such test to work and it's pretty much vanilla gRPC client bootstrap

Reply:
Sure, makes sense.